app/component/StopNearYou.js

```diff
@@ -28,7 +28,8 @@ const StopNearYou = (
     }
   }, [currentTime, currentMode]);
   const description = desc || stop.desc;
-  const isStation = !!stop.parentStation || !!stopId;
+  const isStation =
+    stop.locationType === 'STATION' || !!stop.parentStation || !!stopId;
```

Comment:
I think we can probably just simply check `stop.locationType === 'STATION'` here and if the locationType is missing from query, we can add it to there.